src/test/java/com/uid2/admin/vertx/OperatorKeyServiceTest.java

```diff
@@ -62,14 +78,21 @@ void operatorAdd(Vertx vertx, VertxTestContext testContext) {
     @Test
     void operatorAddUnknownSiteId(Vertx vertx, VertxTestContext testContext) {
         fakeAuth(Role.OPERATOR_MANAGER);
-        post(vertx, "api/operator/add?name=test_client&protocol=trusted&site_id=5", "", expectHttpError(testContext, 400));
+        post(vertx, "api/operator/add?name=test_client&protocol=trusted&site_id=5&roles=optout", "", expectHttpError(testContext, 400));
+    }
+
+    @Test
+    void operatorKeyAddInvalidRole(Vertx vertx, VertxTestContext testContext) {
```

Comment:
After adding the null role validation, add another test case that includes one valid role and one null role